EV_MULTIPLICITY=0 is dropped because libev provided in debian package
is compiled with a different, and legitimate, value.
See messages at [0] and commit [1]
[0]
http://groups.google.com/group/nodejs/search?group=nodejs&q=ev_multiplicity
[1]
http://github.com/ry/node/commit/ffb4b47367cab304ec947ec4ebb0a6c0dc8028b7

--- a/tools/wafadmin/Tools/node_addon.py
+++ b/tools/wafadmin/Tools/node_addon.py
@@ -28,9 +28,8 @@
   lib = join(prefix, 'lib')
 
   conf.env['LIBPATH_NODE'] = lib
-  conf.env['CPPPATH_NODE'] = join(prefix, 'include', 'node')
+  conf.env['CPPPATH_NODE'] = join(prefix, 'include', 'nodejs')
   conf.env['CPPFLAGS_NODE'] = '-D_GNU_SOURCE'
-  conf.env['CPPFLAGS_NODE'] = '-DEV_MULTIPLICITY=0'
 
   # with symbols
   conf.env.append_value('CCFLAGS', ['-g'])
@@ -57,7 +56,7 @@
     nodePath = None
     if not os.environ.has_key('NODE_PATH'):
         if not os.environ.has_key('HOME'):
-            nodePath = join(get_prefix(), 'lib', 'node')
+            nodePath = join(get_prefix(), 'lib', 'nodejs')
         else:
             nodePath = join(os.environ['HOME'], '.node_libraries')
     else:
@@ -70,4 +69,4 @@
         prefix = os.path.abspath(os.path.join(os.path.dirname(__file__), '..', '..', '..', '..'))
     else:
         prefix = os.environ['PREFIX_NODE']
-    return prefix
\ No newline at end of file
+    return prefix
